frontend/src/pages/Alerts/components/AlertTitleField/AlertTitleField.tsx

```diff
@@ -0,0 +1,21 @@
+import { Form, FormState } from '@highlight-run/ui'
+
+import * as styles from './styles.css'
+
+const AlertTitleField = ({ form }: { form: FormState<any> }) => {
+	return (
+		<Form.Input
+			name={String(form.names.name)}
```

Comment:
I'm a little unsure if this is pre-existing in the input, but notice that it always jumps me to the end of the string when I edit the field. For example, below I was trying to change "Alerts" to "Slack Alerts"

Uploading Screen Recording 2023-08-28 at 11.44.16 AM.mov…

Reply:
Good catch, did some more digging on this and resolve it by setting the value field too, since this is all part of the form state.